docs/reference/graph/explore.asciidoc

```diff
@@ -30,6 +30,7 @@ enable you to _spider out_ from one more vertices of interest. You can exclude
 vertices that have already been returned.
 
 [float]
+[role="child_attributes"]
 === Request Body
```

Comment:
Unlike most of our API docs, this one uses H3s.
For some reason, it's applying the `[role="child_attributes"]` attribute to
the H3 rather than its associated div/section.

As a workaround, you can create a section block using `====` and apply the attribute to it.

```suggestion
[float]
=== Request Body

[role="child_attributes"]
====
```